### DISCUSSION TITLE
Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by scottcumings** [community/community#44655] [GitHub Education]
Bug

I have tried to sign up for the student development pack with my school email address; however it is detecting the incorrect school from my email address and I cannot correct it. Is there anyway I can fix this? Thanks

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @scottcumings ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by Daniyal-Aslam** [comment]
Bug

I have tried to sign up for the student development pack with my school email address; however it is detecting the incorrect school from my email address and I cannot correct it. Is there anyway I can fix this? Thanks
